pytesmo/temporal_matching.py

```diff
@@ -3,7 +3,7 @@
 """
 
 import numpy as np
-from scipy.spatial import cKDTree
+from pykdtree.kdtree import KDTree

```

Comment:
please state in the release notes why we have changed from scipy to pykdtree

Reply:
it's faster